hedera-node/src/main/java/com/hedera/services/state/expiry/renewal/RenewalHelper.java

```diff
@@ -140,11 +141,11 @@ public ExpiredEntityClassification classify(long candidateNum, long now) {
 		}
 
 		Pair<List<EntityId>, List<CurrencyAdjustments>> displacements = Pair.of(new ArrayList<>(), new ArrayList<>());
-		final var lastClassifiedTokens = lastClassifiedAccount.tokens();
-		if (lastClassifiedTokens.numAssociations() > 0) {
+		final var associatedTokens = getAssociatedTokenIds(tokenRels.get(), lastClassifiedAccount, Integer.MAX_VALUE);
```

Comment:
Is it okay that the max number of tokens now is going to be 2^31 - 1 ? Would it be better to use a long ?

Reply:
As we get closer to enabling auto-renew/remove for accounts, we will want to re-factor this to do just a handful of treasury returns per renewal cycle; but we don't need to worry about it in this PR imo.